client/me/profile-links/index.jsx

```diff
@@ -173,20 +172,23 @@ class ProfileLinks extends React.Component {
 
 	render() {
 		return (
-			<div>
-				<QueryProfileLinks />
-				<SectionHeader label={ this.props.translate( 'Profile Links' ) }>
-					<AddProfileLinksButtons
-						showingForm={ this.state.showingForm }
-						onShowAddOther={ this.showAddOther }
-						showPopoverMenu={ this.state.showPopoverMenu }
-						onShowAddWordPress={ this.showAddWordPress }
-						onShowPopoverMenu={ this.showPopoverMenu }
-						onClosePopoverMenu={ this.closePopoverMenu }
-					/>
-				</SectionHeader>
-				<Card>{ this.state.showingForm ? this.renderForm() : this.renderProfileLinks() }</Card>
-			</div>
+			<Fragment>
+				<div>
+					<QueryProfileLinks />
+					<SectionHeader label={ this.props.translate( 'Profile Links' ) }>
+						<AddProfileLinksButtons
+							showingForm={ !! this.state.showingForm }
```

Comment:
Is there any benefit from this `!!`?

Reply:
Not sure how that got there tbh. 🤔 

Oh well, nice spot, addressed in the last commit!